base/stats.go

```diff
@@ -1878,6 +1876,10 @@ func (d *DbStats) initDatabaseStats() error {
 	if err != nil {
 		return err
 	}
+	resUtil.TombstoneCount, err = NewIntStat(SubsystemDatabaseKey, "tombstone_count", StatUnitNoUnits, TombstoneCount, StatAddedVersion4dot0dot0, StatDeprecatedVersionNotDeprecated, StatStabilityInternal, labelKeys, labelVals, prometheus.CounterValue, 0)
```

Comment:
Change to a committed stat - we want customers to be able to use this and documenting it as committed will ensure we don't remove this unintentionally.

```suggestion
	resUtil.TombstoneCount, err = NewIntStat(SubsystemDatabaseKey, "tombstone_count", StatUnitNoUnits, TombstoneCount, StatAddedVersion4dot0dot0, StatDeprecatedVersionNotDeprecated, StatStabilityCommitted, labelKeys, labelVals, prometheus.CounterValue, 0)
```

Reply:
done